mesos/src/main/java/tachyon/mesos/TestWorkerExecutor.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.mesos;
+
+import tachyon.Format;
+import tachyon.worker.TachyonWorker;
+
+public class TestWorkerExecutor {
+  public static void main(String[] args) throws Exception {
+    Format.main(new String[] {"worker"});
+    TachyonWorker.main(new String[] {});
+  }
+}
```

Comment:
I'm not sure about the purpose of `TestMasterExecutor` and `TestWorkerExecutor`? I think they are just for proof of concept, do they need to be merged?